### PR TITLE
Primarily add some missing librdmacm man pages

### PR DESCRIPTION
**Pull request by hnrose**
Needed for Debian release

Also, a couple of trivial fixes:
Fix transfer_count default in rstream man page
Add retries option to usage in cmtime example

### CODE REVIEW

**Comment by hnrose** [comment]
@shefty 
Would you review these changes ?
Thanks.

File: librdmacm/man/udpong.1 [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+udpong \- unreliable datagram streaming over RDMA ping-pong test.
+.SH SYNOPSIS
+.sp
+.nf
+\fIudpong\fR [-s server_address] [-b bind_address]
+			[-B buffer_size] [-C transfer_count]
+			[-S transfer_size] [-p server_port] [-T test_option]
+.fi
+.SH "DESCRIPTION"
+Uses unreliable datagram streaming over RDMA protocol (rsocket) to
```
Comment (shefty) [inline librdmacm/man/udpong.1]: I'd drop the word 'streaming' here.  It's confusing as this is a pingpong test.

File: librdmacm/man/udpong.1 [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+.P
+b | blocking - uses blocking calls
+.P
+n | nonblocking - uses non-blocking calls
+.P
+e | echo - server echoes all messages
+.SH "NOTES"
+Basic usage is to start udpong on a server system, then run
+udpong -s server_name on a client system.  udpong
+will run a series of latency and bandwidth performance tests.
```
Comment (shefty) [inline librdmacm/man/udpong.1]: wait - does this do streaming datagram transfers too?  LOL - I can't remember what I wrote for this test anymore.

**Comment by shefty** [comment]
After reading the udpong code to remind myself what it does, the man pages look good. 👍

**Comment by hnrose** [comment]
Yup, it' been quite a while...

I saw that udpong uses rsocket calls ;-)

**Comment by jgunthorpe** [comment]
@hnrose Please send a patch to update the rpm spec files as well..

**Comment by hnrose** [comment]
WIll do shortly

**Pull request merged**